azerothcore/azerothcore-wotlk
# fix(Core/Scripts): Fix bool script hooks not executing code

## Changes Proposed:
-  Code was returned before the script hooks were actually called, preventing scripts from executing.
-  

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/2460

## SOURCE:


## Tests Performed:

- Tested in-game
- 


## How to Test the Changes:


1. Go to Felwood Corrupted Flowers
2. Turn the quest in
3. Notice they despawn and a new plant spawns now

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
